vision/cnns/pytorch/train/benchmarks.yml

```diff
@@ -16,7 +16,7 @@ common_options: &common_options
     - [loss, "loss"]
   env:
     POPLAR_ENGINE_OPTIONS: '{"opt.enableMultiAccessCopies":"false"}'
-    PYTORCH_CACHE_DIR: "./pt_cache/"
+    PYTORCH_CACHE_DIR: "./tmp/pt_cache/"
```

Comment:
is this intended? or should be /tmp/pt_cache?

Reply:
thanks, fixed